src/components/Section.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+interface SectionProps {
+  backgroundColor?: string;
+  children?: ReactNode;
+}
+
+const StyledSection = styled(({ backgroundColor, ...rest }) => <div {...rest} />)`
```

Comment:
evt `StyledDiv`/`StyledBackgroundDiv`
```suggestion
const StyledSection = styled(({ backgroundColor, ...rest }) => <section {...rest} />)`
```

Reply:
litt usikker på om det er riktigst å bruke div eller section her. https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div